components/builder-web/app/origin/origin-page/origin-integrations-tab/_origin-integrations-tab.component.scss

```diff
@@ -16,42 +16,107 @@
     margin: 0 auto rem(30);
   }
 
-  ul {
-    margin-top: 18px;
-    border-top: 1px solid $very-light-gray;
+  .registries {
+    .mat-grid-tile {
+      overflow: initial;
```

Comment:
This isn't supported at all in IE -- did you just need to unset it? https://developer.mozilla.org/en-US/docs/Web/CSS/initial